graphql/api/src/main/java/com/expediagroup/streamplatform/streamregistry/graphql/model/inputs/ProcessInputStreamInput.java

```diff
@@ -25,10 +25,10 @@
 @Value
 @Builder
 public class ProcessInputStreamInput {
-  StreamKeyInput stream;
+  StreamKeyInput streamKeyInput;
```

Comment:
This seems to be the convention we are following in the other input classes?
```suggestion
  StreamKeyInput streamKey;
```